src/site/content/en/blog/iframe-lazy-loading/index.md

```diff
@@ -0,0 +1,305 @@
+
+```html
+<iframe src="https://open.spotify.com/embed/album/1DFixLWuPkv3KT3TnV35m3" 
+        loading="lazy"
+        width="300" 
+        height="380" 
+        frameborder="0" 
+        allowtransparency="true" 
+        allow="encrypted-media"></iframe>
+```
+
+Although the above embeds illustrate the potential benefits to lazy-loading
+iframes for media content, there's the potential to also see these benefits for
+ads.  
+
+### Case study: Natively lazy-loading the Facebook's social plugins
+
+Facebook's _social plugins_ allow developers to embed Facebook content in their
+web pages. There's a number of these plugins offered, such as embedded posts,
+photos, videos, comments… The most popular is the [Like
+plugin](https://developers.facebook.com/docs/plugins/like-button/) - a button
+that shows a count of who has "liked" the page. By default, embedding the Like
+plugin in a webpage (using the FB JSSDK) pulls in ~215KB of resources, 197KB of
+which is JavaScript. In many cases, the plugin may appear at the end of an
+article or near the end of a page, so loading it eagerly when it's offscreen
+may be suboptimal.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="./fblike.png" alt="Facebook Like Button">
+</figure>
+
+Thanks to engineer Stoyan Stefanov, [all of Facebook's social plugins now
+support native iframe
+lazy-loading](https://developers.facebook.com/docs/plugins/like-button#settings).
+Developers who opt in to lazy-loading via the plugins' `data-lazy`
+configuration will now be able to avoid it loading until the user scrolls
+nearby. This enables the embed to still fully function for users that need it,
+while offering data-savings for those who are not scrolling all the way down a
+page. We are hopeful this is the first of many embeds to explore native iframe
+lazy-loading in production.
+
+### Wait, can't browsers just automatically lazy-load offscreen iframes?
+
+They certainly can. In Chrome 77, Chrome added support for automatically
+natively lazy-loading offscreen images and iframes when a user has opted into
+[Lite Mode](https://blog.chromium.org/2019/04/data-saver-is-now-lite-mode.html)
+(Data Saver mode) in Chrome for Android. 
+
+Lite Mode users are commonly in regions of the world where network connection
+quality and data savings plans are not the greatest. Every byte matters and so
+lazy-loading iframes has the potential to make a meaningful difference for
+these users.
+
+Origins can detect what percentage of their traffic is coming from Lite Mode
+users by checking the `navigator.connection.saveData` JavaScript API.
+
+### Can I lazy-load iframes cross-browser? Yes
+
+It is also possible to lazy-load offscreen iframes using the
+[lazysizes](/use-lazysizes-to-lazyload-images/) library. 
+
+This may be desirable if you:
+
+*   require more custom lazy-loading thresholds than what native lazy-loading
+currently offers
+*   are implementing lazy-loading cross-browser
```

Comment:
Don't make cross-browser sound like a choice. There's one web platform.

Reply:
Do you have a suggestion to reword this? The nuance we are trying to convey is:

- Native lazy-loading offers lazy-loading in a way that can be applied as a progressive enhancement. iframes with `loading=lazy` will continue to load per normal in browsers without native lazy-loading support
- Folks that want to have this behavior cross-browser until it is widely supported have options like JS libraries.

Perhaps I can capture that nuance better here.

Reply:
The way you have said it just now looks pretty good. Using the phrase 'progressive enhancement' presumes support for common platform.

Reply:
Cool! Updated in https://github.com/GoogleChrome/web.dev/pull/3536/commits/d377edb419665320b57fe4052948f48407e3ce2a